src/redis/BetterSocialCronQueue.js

```diff
@@ -11,7 +11,7 @@ const {scoringDailyProcessJob} = require('../processes/scoring-daily-process');
 const {EVENT_DAILY_PROCESS_TRIGGER} = require('../processes/scoring-constant');
 
 /**
- * @typedef {"dailyCredderUpdate" | "dailyDeleteExpiredPost" | "dailyRssUpdate" | "dailyScoring" | "hourlyRefreshMaterializedView"} CronFlag
+ * @typedef {"dailyCredderUpdate" | "dailyDeleteExpiredPost" | "dailyRssUpdate" | "dailyScoring" | "topicAutoMessage" | "hourlyRefreshMaterializedView"} CronFlag
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the "topicAutoMessage" flag to the CronFlag typedef is a good change, as it allows for the new feature to be scheduled and run at regular intervals. This change is consistent with the existing code structure and conventions.
```diff
- * @typedef {"dailyCredderUpdate" | "dailyDeleteExpiredPost" | "dailyRssUpdate" | "dailyScoring" | "hourlyRefreshMaterializedView"} CronFlag
+ * @typedef {"dailyCredderUpdate" | "dailyDeleteExpiredPost" | "dailyRssUpdate" | "dailyScoring" | "topicAutoMessage" | "hourlyRefreshMaterializedView"} CronFlag
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->